sympy/series/tests/test_order.py

```diff
@@ -441,3 +441,6 @@ def test_issue_15539():
 
 def test_issue_18606():
     assert unchanged(Order, 0)
+
+def test_issue_9910():
+    assert O(x*log(x)+sin(x), (x, oo)) == O(x*log(x), (x, oo))
```

Comment:
Please add spaces around `+`.
Moreover, it will be better if the testcase is added closer to testcases having similar issue numbers and not at the end of the file.
Apart from that everything looks fine.

Reply:
Also please maintain two line spacing between the testcases.